pkg/instancegroups/instancegroups.go

```diff
@@ -65,6 +67,32 @@ func NewRollingUpdateInstanceGroup(cloud fi.Cloud, cloudGroup *cloudinstances.Cl
 	}, nil
 }
 
+// User input routine copied from vendor/google.golang.org/api/examples/gmail.go
+func PromptInteractive(upgradedHost string) (stop_prompting bool) {
+	stop_prompting = false
+	reader := bufio.NewReader(os.Stdin)
+	glog.Infof("Pausing after finished %q", upgradedHost)
+	fmt.Printf("Continue? (Y)es, (N)o, (A)lwaysYes: [Y] ")
+	val := ""
+	var err error
+	if val, err = reader.ReadString('\n'); err != nil {
+		glog.Fatalf("unable to interpret input: %v", err)
+	}
+	val = strings.TrimSpace(val)
+	val = strings.ToLower(val)
+	switch val {
+	case "y", "", "\n":
+		glog.V(4).Infof("Continuing with next host (response %q)\n", val)
```

Comment:
fmt.Printf?

Reply:
I don't quite understand this comment.  I was attempting to make a log message that only shows up for higher verbosity levels.  Are you saying that you'd rather no output for this?

Now I'm wondering if I should just omit the test for "y" altogether.  "n" means stop, "a" means no more prompts, anything else should be treated as "y".  If you won't want any glog output, I can just omit that whole case "y" section and it should behave the same.

Reply:
I ended up just completely getting rid of the case "y" section.  Only [nN] or [aA] make something different happen (stop and no more prompting, respectively).  Anything else is a default "y".